gui/wxpython/gui_core/mapdisp.py

```diff
@@ -720,7 +720,7 @@ def SetBindRegions(self, on):
                 self.firstMapWindow.zoomChanged.connect(self.OnZoomChangedFirstMap)
             else:
                 self.secondMapWindow.zoomChanged.connect(self.OnZoomChangedSecondMap)
-        else:
+        else:  # noqa: PLR5501
```

Comment:
As above, could be probably included in the changeset...

Reply:
The pattern (two binary conditions) is evident without collapsing

Reply:
Couldn't find a better way here. Visually similar, but not the same functions called (connect vs disconnect).